test/commands/transaction/create.test.js

```diff
@@ -55,36 +55,75 @@ describe('transaction:create', () => {
 
```

Comment:
Maybe also switch order in imports and `setupTest` defintion for consistency.

Reply:
Also minor fix for test/commands/transaction/create/vote.test.js (not in this PR but might as well fix here): Test title `transaction:create:transfer` should be `transaction:create:vote`